### DISCUSSION TITLE
window manager download

### DISCUSSION THREAD

**Discussion by ghost** [courses-ionio/hci#1978] [Q&A] [answered]
Στις οδηγίες αναφέρεται οτι μπορουμε να κατεβάσουμε ενα window manager. Αντί για το i3 ειμαι εξοικειωμένος με το [hyprland](https://hyprland.org/), είναι αποδεκτό; @epidrome

**Comment by epidrome** [comment, marked as answer]
@vmillios ωραίο φαίνεται, φαίνεται αποδεκτό, αλλά δεν το γνωρίζω και θα ήθελα να μάθω περισσότερα όταν έρθεις στην συνάντηση
